test/conformance/config/ptptestconfig.yaml

```diff
@@ -3,16 +3,24 @@ global:
  maxoffset: -100
  minoffset: 100
 soaktest:
-  enable: false
-  masteroffset:
-   enable: true
-   failfast: true
-   duration: 10
-  testa:
-   enable: true
-   failfast: true
-   duration: 10
-  testb:
-   enable: true
-   failfast: true
-   duration: 10
+  disable_all: false
+  duration: 10
+  failure_threshold: 2
+  master_offset:
+    spec: 
+      enable: true
+      duration: 10
```

Comment:
Since global threshold and this one is same, if I don't have this set then it will use global threshold  value right ?

Reply:
So when you load the config you should be filling in the local variables from global if missing else use default for everything. . Can you have that added to your unit test .

Reply:
> Since global threshold and this one is same, if I don't have this set then it will use global threshold value right ?

Yes!

Reply:
yes